LayoutTests/TestExpectations

```diff
@@ -7381,6 +7381,7 @@ imported/w3c/web-platform-tests/dom/events/scrolling/wheel-event-transactions-ta
 
 imported/w3c/web-platform-tests/css/css-color/deprecated-sameas-002.html [ ImageOnlyFailure ]
 
+http/tests/site-isolation/page-zoom.html [ Skip ]
```

Comment:
why do we skip this? is this something you want to investigate on a separated PR? If so can you add the bugzilla link here?